docs/usage/caching.rst

```diff
@@ -15,59 +15,63 @@ expected. Litestar comes with a simple mechanism for caching:
    @get("/cached-path", cache=True)
    def my_cached_handler() -> str: ...
 
-By setting ``cache=True`` in the route handler, caching for the route handler will be enabled for the
-:attr:`ResponseCacheConfig.default_expiration <.config.response_cache.ResponseCacheConfig.default_expiration>`.
+By setting :paramref:`~litestar.handlers.HTTPRouteHandler.cache` to ``True``, the response from the handler
+will be cached. If no ``cache_key_builder`` is set in the route handler, caching for the route handler will be
+enabled for the :attr:`~.config.response_cache.ResponseCacheConfig.default_expiration`.
```

Comment:
What's the relationship between the `cache_key_builder` being defined and the caching being enabled for `default_expiration`?

The way that this reads is that if we set `cache=True` and provide a `cache_key_builder` then something other than enabling caching with the default expiration time would occur. Is that correct?